cmd/oras/root/push.go

```diff
@@ -224,19 +233,40 @@ func runPush(ctx context.Context, opts pushOptions) error {
 	return opts.ExportManifest(ctx, memoryStore, root)
 }
 
-func updateDisplayOption(opts *oras.CopyGraphOptions, fetcher content.Fetcher, verbose bool) {
+func doPush(pack packFunc, copy copyFunc, dst oras.Target) (ocispec.Descriptor, error) {
```

Comment:
nit: code consistency against `pushArtifact`
```suggestion
func doPush(dst oras.Target, pack packFunc, copy copyFunc) (ocispec.Descriptor, error) {
```